homeassistant/components/sensor/discogs.py

```diff
@@ -20,18 +22,41 @@
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_IDENTITY = 'identity'
-
 CONF_ATTRIBUTION = "Data provided by Discogs"
 
 DEFAULT_NAME = 'Discogs'
 
-ICON = 'mdi:album'
-
-SCAN_INTERVAL = timedelta(hours=2)
+ICON_RECORD = 'mdi:album'
+ICON_PLAYER = 'mdi:record-player'
+UNIT_RECORDS = 'records'
+
+SCAN_INTERVAL = timedelta(minutes=10)
+
+LIST_MONITORED_CONDITIONS = ['collection', 'wantlist', 'random_record']
+
+SENSORS = {
+    'collection': {
+        'name': 'Collection',
+        'icon': 'mdi:album',
+        'unit_of_measurement': 'records'
+    },
+    'wantlist': {
+        'name': 'Wantlist',
+        'icon': 'mdi:album',
+        'unit_of_measurement': 'records'
+    },
+    'random_record': {
+        'name': 'Random Record',
+        'icon': 'mdi:record_player',
+        'unit_of_measurement': None
+    },
+}
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_TOKEN): cv.string,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_MONITORED_CONDITIONS, default=LIST_MONITORED_CONDITIONS):
```

Comment:
`default=list(SENSORS)`

Reply:
Not sure, will this not also include the properties on the object, when I only want the keys (like in LIST_MONITORED_CONDITIONS)?

Reply:
Good, thanks! Will correct this.